File: C/C - Programmieren für Einsteiger/uebung2b.c
/*
2.  Gestalten Sie ein Programm, das vom Anwender zwei Fließkommazahlen abfragt.
    Geben Sie als Ergebnis die Summe der beiden Werte aus.

Neumann, Markus. C Programmieren: für Einsteiger: Der leichte Weg zum C-Experten (Einfach Programmieren lernen 8) (German Edition) (S.55). Kindle-Version.
*/

#include <stdio.h>

void main()
{
    float x;
    float y;

    printf ("Geben Sie den ersten Wert ein: ");
    scanf ("%f", &x);
    printf ("Geben Sie den ersten Wert ein: ");
    scanf ("%f", &y);
    printf ("Ergebnis: %f", x + y);
}

File: Python/Python - Programmieren für Einsteiger/uebung2b.py
"""
2.    Schreiben Sie ein Programm, das den Anwender dazu auffordert, einen beliebigen Inhalt einzugeben.
      Speichern Sie den Wert in einer Variablen und geben Sie anschließend ihren Datentyp aus.

Bonacina, Michael. Python 3: Programmieren für Einsteiger: Der leichte Weg zum Python-Experten (Einfach Programmieren lernen 2) (German Edition) (S.54-55). Kindle-Version.
"""


def main():
    x = eval(input("Geben Sie etwas ein: "))

    print("Type: {}".format(type(x)))

if(__name__ == "__main__"):
    main()